pkg/sensors/tracing/generickprobe.go

```diff
@@ -454,6 +469,45 @@ func createGenericKprobeSensor(name string, kprobes []v1alpha1.KProbeSpec) (*sen
 		logger.GetLogger().Infof("Added generic kprobe sensor: %s -> %s", load.Name, load.Attach)
 	}
 
+	if len(multiIDs) != 0 {
+		pinFile := "multi_kprobe"
```

Comment:
Should we make this `sensors.PathJoin(sensorPath, "multi_kprobe")`? (see other comment as well)